### PR TITLE
refactor: remove settings attribute in the config render

### PR DESCRIPTION
**Pull request by Peefy**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [x] N
- [ ] Y 

refactor: remove settings attribute in the config render and bump expected kclvm version to v0.4.5

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [x] Other

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [x] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [x] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [ ] Other

+ All test cases in the konfig repo

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Comment by howieyuen** [comment]
test job is failed, plz take a look

File: base/pkg/kusion_models/kube/render/render.k [hunk 1]
```diff
@@ -112,7 +109,8 @@ else:
...
                                 for dependOn in __resource_map___[k]
                 ]
-                "attributes" = _resource | __output_inline__
-            } | __output_standalone__ for _resource in _res
+                "attributes" = _resource
+            } for _resource in _res
         ] for _kind, _res in __resource_map___ 
-    } | __output_ignore__
+    }], [])
+    manifests.yaml_stream(__resource__list__)
```
Comment (amyXia1994) [inline base/pkg/kusion_models/kube/render/render.k:116]: seems this line `manifests.yaml_stream(__resource__list__)` can be shared by both the `if` and `else` block

File: base/pkg/kusion_models/kube/render/render.k [hunk 2]
```diff
@@ -83,25 +78,27 @@ __resource_map_original___: {str: []} = {
...
 # structure output
 if option("__konfig_output_format__")?.lower() == "raw":
-    konfig_kubeKubernetes = { 
-        _kind: [_resource | __output_standalone__ for _resource in _res] \
+# Convert resource map into a resource list
+    __resource__list__ = sum([res_list for _, res_list in { 
+        _kind: [_resource for _resource in _res] \
             for _kind, _res in __resource_map___
-    } | __output_ignore__
+    }], [])
```
Comment (amyXia1994) [inline base/pkg/kusion_models/kube/render/render.k:93]: could you please explain why we need to sum the "resource list generated by the list comp" and "[]"(**an empty list**?)

Comment (Peefy) [reply base/pkg/kusion_models/kube/render/render.k]: > could you please explain why we need to sum the "resource list generated by the list comp" and "[]"(**an empty list**?)

add more comments.

File: base/pkg/kusion_models/kube/render/render.k (hunk 1 above)
Comment (Peefy) [reply base/pkg/kusion_models/kube/render/render.k]: > seems this line `manifests.yaml_stream(__resource__list__)` can be shared by both the `if` and `else` block

Have moved them into one kcl function.

**Review by amyXia1994 — APPROVED** [review]
lgtm

**Pull request merged**